compiler/one-cmds/one-init

```diff
@@ -177,6 +196,51 @@ def _get_executable(args, backends_list):
         raise FileNotFoundError(backend_base + ' not found')
 
 
+def _generate():
+    # generate cfg file
+    config = CommentableConfigParser()
+
+    def _add_onecc_sections():
+        pass  # NYI
+
+    def _add_from_backend_cfg():
+        '''
+        This copies key & value in sections of backend cfg into one cfg.
+        Call this after setting default values for circle since there might be values overrided
+        by backend.
+        '''
+        pass  # NYI
+
+    def _gen_import():
+        pass  # NYI
+
+    def _gen_optimize():
+        pass  # NYI
+
+    def _gen_quantize():
+        pass  # NYI
+
+    def _gen_codegen():
```

Comment:
CC @mhs4670go 
@hyunsik-yoon , #9354 is WIP and these may not be compatible with it.
please add a TODO comment to revise this later.

Reply:
Thanks. TODO was added.